### PR TITLE
Gallery adaptions

### PR DESCRIPTION
**Pull request by lior-tamir**
Added metadata to ARM templates
dark+light screenshots
Encapsulated incident trigger/alert trigger in different folders, so gallery can read them as different playbooks
changed azuredeploy.json names (gallery expects "azuredeploy.json" file in each folder)

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by preetikr** [comment]
@mpo-sec - Please check if this aligns with the metadata schema format for the ARM API. Thanks.

File: Playbooks/Block-AADUser/incident-trigger/azuredeploy.json [hunk 1]
```diff
@@ -2,8 +2,18 @@
...
-        "comments": "This playbook will disable the user in Azure Active Directoy and add a comment to the incident",
-        "author": "Nicholas DiCola"
+        "title": "Identity Protection response from Teams", 
+        "description": "For each account entity included in the incident, this playbook will disable the user in Azure Active Directoy and add a comment to the incident",
+        "prerequisites": "",
+        "lastUpdateTime": "2021-05-30T10:00:00.000Z", 
+        "entities": ["Account"], 
+        "tags": ["Remidiation"], 
+        "support": {
+            "kind": "Community"
```
Comment (mpo-sec) [inline Playbooks/Block-AADUser/incident-trigger/azuredeploy.json:12]: this should be `community` to match the metadata enum

File: Playbooks/Confirm-AADRiskyUser/incident-trigger/azuredeploy.json [hunk 2]
```diff
@@ -2,8 +2,18 @@
...
-        "comments": "This playbook will set the Risky User property in AAD using Graph API using a Beta API.  NOTE:  You must create an app registration for graph api with appropriate permissions.  NOTE:  You will need to add the managed identity that is created by the logic app to the Security Administrator role in Azure AD.",
-        "author": "Nicholas DiCola"
+        "title": "Confirm-AADRiskyUser", 
+        "description": "For each account entity included in the incident, this playbook will set the Risky User property in AAD using Graph API using a Beta API.",
+        "prerequisites": "1. You must create an app registration for graph api with appropriate permissions.  2. You will need to add the managed identity that is created by the logic app to the Security Administrator role in Azure AD.",
+        "lastUpdateTime": "2021-05-30T10:00:00.000Z", 
+        "entities": ["Account"], 
+        "tags": ["Remidiation"], 
+        "support": {
+            "kind": "Community"
```
Comment (mpo-sec) [inline Playbooks/Confirm-AADRiskyUser/incident-trigger/azuredeploy.json:12]: same

File: Playbooks/Create-AzureDevOpsTask/incident-trigger/azuredeploy.json [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
     "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Create-AzureDevOpsTask", 
+        "description": "This playbook will create the Azure DevOps task filled with the Azure Sentinel incident details.",
+        "prerequisites": "",
+        "lastUpdateTime": "2021-05-30T10:00:00.000Z", 
+        "entities": [], 
+        "tags": ["Sync"], 
+        "support": {
+            "kind": "Community"
```
Comment (mpo-sec) [inline Playbooks/Create-AzureDevOpsTask/incident-trigger/azuredeploy.json:12]: same

File: Playbooks/Dismiss-AADRiskyUser/incident-trigger/azuredeploy.json [hunk 4]
```diff
@@ -2,8 +2,18 @@
...
-        "comments": "This playbook will dismiss the Risky User property in AAD using Graph API using a Beta API.",
-        "author": "Nicholas DiCola"
+        "title": "Dismiss-AADRiskyUser", 
+        "description": "This playbook will dismiss the Risky User property in AAD using Graph API.",
+        "prerequisites": "",
+        "lastUpdateTime": "2021-05-30T10:00:00.000Z", 
+        "entities": ["Account"], 
+        "tags": ["Remediation"], 
+        "support": {
+            "kind": "Community"
```
Comment (mpo-sec) [inline Playbooks/Dismiss-AADRiskyUser/incident-trigger/azuredeploy.json:12]: same

**Pull request merged**